libsolidity/ast/CallGraph.h

```diff
@@ -61,8 +61,8 @@ struct CallGraph
 	/// any calls.
 	std::map<Node, std::set<Node, CompareByID>, CompareByID> edges;
 
-	/// Contracts that may get created with `new` by functions present in the graph.
-	std::set<ContractDefinition const*, ASTNode::CompareByID> createdContracts;
+	/// Contracts that are referenced & used by this contract
```

Comment:
I'd make it a bit more specific. How about this?
```suggestion
	/// Contracts that would need to be compiled first to make it possible to compile this contract.
	/// This is due to the need to access their bytecode, either directly or indirectly through contract creation at runtime.
```

Reply:
I didn't use your phrasing 1:1 but I changed it to `/// Contracts that need to be compiled before this one can be compiled.`